functions.php

```diff
@@ -7,20 +7,110 @@
 function databaseConnect(): PDO {
     $db = new PDO('mysql:host=db; dbname=collection_project', 'root', 'password');
     $db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+    $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_WARNING);
     return $db;
 }
+
 /**
  * Creates query fetching all data from the database
  *
  * @param PDO created by connection with database
  * @return array returns all of the data requested from the database
  */
-function databaseFetchAll($db): array{
-    $query = $db->prepare("SELECT `charname`, `class`, `level`, `strength`, `dexterity`, `constitution`, `intelligence`, `wisdom`, `charisma` FROM `characters`;");
+function databaseFetchAll(PDO $db): array {
+    $query = $db->prepare("SELECT `charname`, `class`, `level`, `strength`, `dexterity`, `constitution`, `intelligence`, `wisdom`, `charisma`, `id` FROM `characters` WHERE `deleted` = 0;");
```

Comment:
Very minor change, but I'd suggest moving the id to be first just so it matches your editChar query